pypeit/core/telluric.py

```diff
@@ -1220,7 +1261,7 @@ def sensfunc_telluric(wave, counts, counts_ivar, counts_mask, exptime, airmass,
 
     return meta_table, out_table
 
-def create_bal_mask(wave):
+def create_bal_mask(wave, bal_mask):
     """
     Example of a utility function for creating a BAL mask for QSOs with BAL features. Can also be used to mask other
     features that the user does not want to fit.
```

Comment:
Please update the docs and explain how the bal_mask argument is supposed to be used. Also, I would call this something else like bal_wv_min_mx or something getting at its meaning, since it is not actually a mask.

Reply:
Done